List/List.jsx

```diff
@@ -160,13 +121,36 @@ class ListSecondaryText extends ListTextContainer {
   }
 }
 
+class ListGroup extends MaterialComponent {
+  constructor() {
+    super();
+    this.componentName = 'list-group';
+  }
+}
+
+class ListGroupHeader extends MaterialComponent {
```

Comment:
Wouldn't it be better to name it `SubHeader` instead of just `Header`?

Reply:
Nah! We tend to keep names transparent to whatever `mdc-web` has